karma.conf.js

```diff
@@ -1,29 +1,32 @@
 // Karma configuration
 
 module.exports = function(config) {
-  var customLaunchers = {};
-  var browsers = [];
-  var useSauce = false;
+  const customLaunchers = {};
+  let browsers = [];
+  let useSauce = false;
+  let transpile = false;
```

Comment:
We are still transpiling modules, so perhaps this should have a more specific name for clarity.